sdk/iot/hub/samples/src/paho_iot_hub_c2d_example.c

```diff
@@ -109,30 +109,33 @@ static az_result create_mqtt_endpoint(char* destination, int32_t destination_siz
 
 static az_result read_configuration_and_init_client()
 {
+  // Read the user environment variables used to connect to IoT Hub
   az_span cert = AZ_SPAN_FROM_BUFFER(x509_cert_pem_file);
   AZ_RETURN_IF_FAILED(read_configuration_entry(
-      "X509 Certificate PEM Store File", DEVICE_X509_CERT_PEM_FILE, NULL, false, cert, &cert));
+      "X509 Certificate PEM Store File", ENV_DEVICE_X509_CERT_PEM_FILE, NULL, false, cert, &cert));
 
   az_span trusted = AZ_SPAN_FROM_BUFFER(x509_trust_pem_file);
   AZ_RETURN_IF_FAILED(read_configuration_entry(
-      "X509 Trusted PEM Store File", DEVICE_X509_TRUST_PEM_FILE, "", false, trusted, &trusted));
+      "X509 Trusted PEM Store File", ENV_DEVICE_X509_TRUST_PEM_FILE, "", false, trusted, &trusted));
 
   az_span device_id_span = AZ_SPAN_FROM_BUFFER(device_id);
-  AZ_RETURN_IF_FAILED(
-      read_configuration_entry("Device ID", DEVICE_ID, "", false, device_id_span, &device_id_span));
+  AZ_RETURN_IF_FAILED(read_configuration_entry(
+      "Device ID", ENV_DEVICE_ID, "", false, device_id_span, &device_id_span));
 
   az_span iot_hub_hostname_span = AZ_SPAN_FROM_BUFFER(iot_hub_hostname);
   AZ_RETURN_IF_FAILED(read_configuration_entry(
       "IoT Hub Hostname",
-      IOT_HUB_HOSTNAME,
+      ENV_IOT_HUB_HOSTNAME,
       "",
       false,
       iot_hub_hostname_span,
       &iot_hub_hostname_span));
 
+  // Paho requires that the MQTT endpoint be of the form ssl://<HUB ENDPOINT>:8883
   AZ_RETURN_IF_FAILED(
       create_mqtt_endpoint(mqtt_endpoint, (int32_t)sizeof(mqtt_endpoint), iot_hub_hostname_span));
 
+  // Initialize the az_iot_hub_client
```

Comment:
this could be more descriptive.
"Initialize  the hub client with the hub host endpoint with the default connection options"